src/main/java/io/opentracing/contrib/neo4j/TracingTransaction.java

```diff
@@ -118,7 +118,8 @@ public Result run(String statementTemplate) {
   @Override
   public Result run(Query query) {
     Span span = TracingHelper.build("run", parent, tracer);
-    span.setTag(Tags.DB_STATEMENT.getKey(), query.toString());
+    span.setTag(Tags.DB_STATEMENT.getKey(), query.text());
```

Comment:
Here I add query and parameters as a separate tags.
`query.getParameters()` is always not null